src/secp256k1.c

```diff
@@ -517,7 +517,8 @@ static int secp256k1_ecdsa_sign_inner(const secp256k1_context* ctx, secp256k1_sc
     }
     /* sign-to-contract commitments only work with the default nonce function,
      * because we need to ensure that s2c_data is actually hashed into the nonce and
-     * not just ignored. */
+     * not just ignored. Otherwise an attacker can exfiltrate the secret key by
+     * signing the same message twice with different commitments. */
```

Comment:
twice? ;)

Reply:
Lol, fine, do we agree that you can do it with three sigs at least?